extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -208,6 +208,11 @@ class AmpLightbox extends AMP.BaseElement {
    * @private
    */
   scrollHandler_() {
+    // If scroll top is 0, it's set to 1 to avoid scroll-freeze issue.
+    if (this.element./*OK*/scrollTop == 0) {
+      this.element./*OK*/scrollTop = 1;
+      this.update_(1);
```

Comment:
no need to call `update()`, because this.pos_ is already updated due to `this.pos_ = currentScrollTop;` (two lines below)